Allow specifying the Makefile's python interpreter

The bundled loadtest doesn't build on pypy currently (due to loads/psutil), a quick workaround to specify cpython for the Makefile's venv should workaround this for now:

```
pjenvey@ubuntu:~/src/moz/tokenserver/loadtest$ make
ARCHFLAGS=-Wno-error=unused-command-line-argument-hard-error-in-future ../local/bin/pip install pexpect
DEPRECATION: pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.
Collecting pexpect
  Downloading pexpect-4.8.0-py2.py3-none-any.whl (59 kB)
     |████████████████████████████████| 59 kB 1.5 MB/s 
Collecting ptyprocess>=0.5
  Downloading ptyprocess-0.6.0-py2.py3-none-any.whl (39 kB)
Installing collected packages: ptyprocess, pexpect
Successfully installed pexpect-4.8.0 ptyprocess-0.6.0
ARCHFLAGS=-Wno-error=unused-command-line-argument-hard-error-in-future ../local/bin/pip install https://github.com/mozilla-services/loads/archive/master.zip
DEPRECATION: pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support pip 21.0 will remove support for this functionality.
Collecting https://github.com/mozilla-services/loads/archive/master.zip
  Downloading https://github.com/mozilla-services/loads/archive/master.zip
     | 1.3 MB 719 kB/s
Collecting pyzmq
  Downloading pyzmq-19.0.2.tar.gz (1.2 MB)
     |████████████████████████████████| 1.2 MB 3.0 MB/s 
Collecting psutil
  Downloading psutil-5.7.2.tar.gz (460 kB)
     |████████████████████████████████| 460 kB 28.8 MB/s 
    ERROR: Command errored out with exit status 1:
     command: /media/ext/pjenvey/src/moz/tokenserver/local/bin/pypy -c 'import sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-LySEBr/psutil/setup.py'"'"'; __file__='"'"'/tmp/pip-install-LySEBr/psutil/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-zSjOyd
         cwd: /tmp/pip-install-LySEBr/psutil/
    Complete output (7 lines):
    Traceback (most recent call last):
      File "<module>", line 1, in <module>
      File "/tmp/pip-install-LySEBr/psutil/setup.py", line 46, in <module>
        from _compat import PY3  # NOQA
      File "/tmp/pip-install-LySEBr/psutil/psutil/_compat.py", line 199, in <module>
        "broken or incompatible Python implementation, see: "
    RuntimeError: broken or incompatible Python implementation, see: https://github.com/giampaolo/psutil/issues/1659
    ----------------------------------------
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
Makefile:12: recipe for target 'build' failed
make: *** [build] Error 1

```

--- a/Makefile
+++ b/Makefile
@@ -1,5 +1,6 @@
 VIRTUALENV = virtualenv
 VENV := $(shell echo $${VIRTUAL_ENV-local})
+PTYPE = pypy
 PYTHON = $(VENV)/bin/python
 NOSE = $(VENV)/bin/nosetests
 DEV_STAMP = $(VENV)/.dev_env_installed.stamp
@@ -48,11 +49,11 @@ virtualenv: $(PYTHON)
 $(PYTHON):
 	# The latest `pip` doesn't work with pypy 2.7 on some platforms.
 	# Pin to a working version; ref https://github.com/pypa/pip/issues/8653
-	$(VIRTUALENV) -p pypy --no-pip $(VENV)
+	$(VIRTUALENV) -p $(PTYPE) --no-pip $(VENV)
 	$(VENV)/bin/easy_install pip==20.1.1
 
 build-requirements:
-	$(VIRTUALENV) -p pypy --no-pip $(TEMPDIR)
+	$(VIRTUALENV) -p $(PTYPE) --no-pip $(TEMPDIR)
 	$(TEMPDIR)/bin/easy_install pip==20.1.1
 	ARCHFLAGS=$(ARCHFLAGS) $(TEMPDIR)/bin/pip install -Ue .
 	$(TEMPDIR)/bin/pip freeze | grep -v -- '^-e' > requirements.txt

